models/balancer/arbitrum/balancer_v2_arbitrum_pools_fees.sql

```diff
@@ -9,36 +10,38 @@
         post_hook='{{ expose_spells(\'["arbitrum"]\',
                                     "project",
                                     "balancer_v2",
-                                    \'["metacrypto", "jacektrocinski"]\') }}'
+                                    \'["metacrypto", "jacektrocinski", "thetroyharris"]\') }}'
     ) 
 }}
 
 {% set event_signature = '0xa9ba3ffe0b6c366b81232caab38605a0699ad5398d6cce76f91ee809e322dafc' %}
 {% set project_start_date = '2021-08-26' %}
 
 WITH registered_pools AS (
-    SELECT DISTINCT
-        `poolAddress` AS pool_address
+    SELECT
+        DISTINCT poolAddress AS pool_address
     FROM
         {{ source ('balancer_v2_arbitrum', 'Vault_evt_PoolRegistered') }}
+    {% if is_incremental() %}
+    WHERE evt_block_time >= DATE_TRUNC('day', NOW() - interval '7' day)
+    {% endif %}
 )
 SELECT
+    'arbitrum' AS blockchain,
     logs.contract_address,
     logs.tx_hash,
     logs.tx_index,
     logs.index,
     logs.block_time,
     logs.block_number,
-    bytea2numeric_v3 (SUBSTRING(logs.data FROM 32 FOR 64)) * 1 AS swap_fee_percentage
+    bytea2numeric(bytearray_ltrim(logs.data)) AS swap_fee_percentage
```

Comment:
`byte2numeric` should no longer be needed on dunesql. you can find a good replacement here:
https://dune.com/docs/query/DuneSQL-reference/Functions-and-operators/varbinary/

Reply:
Perfect, I have changed it. However, I end up casting it to double to keep the original data type.